src/components/ScriptureTable.tsx

```diff
@@ -1286,7 +1315,6 @@ export function ScriptureTable(props: IProps) {
           mediaRemoteIds[0]
       );
     }
-    setDoSave(true);
```

Comment:
Not sure why we don't want to save after upload?

Reply:
Everything is saved as we go.